lib/tasks/deploy.rake

```diff
@@ -0,0 +1,33 @@
+namespace :deploy do
+  namespace :cascade do
+    desc "Deploy cascade-assets to Cascade"
+    task all: :environment do
```

Comment:
How is this called from the command line?

Reply:
`rake deploy:cascade:all RAILS_ENV=development` will run all steps of the deploy process. I created each part of the deploy process as a separate task,  for example: `rake deploy:cascade:asset_block` for testing purposes. This can be changed in the final cap script if you want.